service/csi_extension_server.go

```diff
@@ -304,7 +304,7 @@ func (s *service) checkIdempotency(ctx context.Context, snapshotsToMake *siotype
 			consistencyGroupMap[existingSnap.Name] = ""
 			//a snapshot in snapshotsToMake already exists in array, update maps
 			foundGrpID := systemID + "-" + existingSnap.ConsistencyGroupID
-			if snap.VolumeID == existingSnap.AncestorVolumeID && snapGrpID == foundGrpID {
+			if snap.VolumeID == existingSnap.AncestorVolumeID && systemID+"-"+snapGrpID == foundGrpID {
```

Comment:
add extra spaces for readability ?

Reply:
it's a gofmt error if I do that in an if statement